build_darwin_x86_64.sh

```diff
@@ -47,10 +47,9 @@ gpg --import gpg-keys/libevent.gpg && \
 gpg "libevent-$LIBEVENT_VERSION.tar.gz.asc" && \
 echo "$LIBEVENT_HASH  libevent-$LIBEVENT_VERSION.tar.gz" | shasum -a 256 -c - && \
 tar -zxvf "libevent-$LIBEVENT_VERSION.tar.gz" -C x86_64 && \
-cp patch/libevent/test/regress_bufferevent.c "x86_64/libevent-$LIBEVENT_VERSION/test/regress_bufferevent.c" && \
```

Comment:
If we're dropping the patch on macos x64, we should probably also drop it on macos arm64 and remove patch/libevent/test/regress_bufferevent.c from the repo, no?

Reply:
the macos builds are broken for unrelated reasons; once fixed in #78 i'll pull this into that PR